recipes/npy-append-array/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "npy-append-array" %}
+{% set version = "0.9.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/npy-append-array-{{ version }}.tar.gz
+  sha256: a826468d20a69653f12ed791553da8753ea3bae88a3b165722f8b069b2b86461
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - numpy
+
+test:
+  imports:
+    - npy_append_array
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/xor2k/npy-append-array
+  summary: Create Numpy `.npy` files that are larger than the main memory
+  license: MIT
+  license_file: LICENSE
```

Comment:
Add comment:

```yaml
# License file manually packaged: See https://github.com/xor2k/npy-append-array/pull/1
license_file: LICENSE
```

I opened a PR to fix the need to manually package the license file in future in future releases (https://github.com/xor2k/npy-append-array/pull/1). The comment is a visual reminder to check the status of this in the future and to remove the manually packaged version when the license file is included in the source distribution.